lib/src/client.dart

```diff
@@ -2098,14 +2098,22 @@ class Client extends MatrixApi {
   /// Resets all settings and stops the synchronisation.
   Future<void> clear() async {
     Logs().outputEvents.clear();
+    DatabaseApi? legacyDatabase;
```

Comment:
lg, can you add a test for this? should be fairly straight forward

Reply:
@coder-with-a-bushido PR looks stuck ping :3

Reply:
@td-famedly pls check now